docs/acr-roadmap.md

```diff
@@ -1,117 +1,159 @@
 
-## Completed Items ##
-Since posting our backlog, we wanted to provide an update on completed work, and updated priorities.
+## Backlog
 
-### Helm Chart Repos
+The following items are in our backlog. Some are in active development. The majority of our backlog items source from users, via [uservoice][uservoice].
 
-ACR now supports pulling and pushing [Helm Charts](https://Helm.sh) directly to your Azure Container Registry. By using ACR, charts are secured and geo-replicated alongside the images they reference. 
 
-To get started: [ACR Helm Chart Repo documentation](https://aka.ms/acr/helm-repos)
+* [Vulnerability Scanning Updates](#vulnerability-scanning-updates)
+* [Limit Endpoint Access /VNet Support](#limit-endpoint-access-vnet-support)
+* [When will ACR VNet support GA](#when-will-acr-vnet-support-ga)
+* [Tokens & Repository Scoped Permissions (RBAC)](#tokens--repository-scoped-permissions-rbac)
+* [Diagnostics & Monitoring](#diagnostics--monitoring)
+* [Health Checks](#health-checks)
+* [Auto Purge](#auto-purge)
+* [ACR Helm GA](#acr-helm-ga)
+* [Performance & Scalability](#performance--scalability)
+* [Container Build Caching with ACR Tasks and BuildX](#container-build-caching-with-acr-tasks-and-buildx)
+* [Helping with Prioritization](#helping-with-prioritization)
 
-### The Donut is Dead
+### Vulnerability Scanning Updates
 
-ACR Registries used to have upper limits on the amount of images that could be pushed to each registry. While we [removed this constraint](https://stevelasker.blog/2018/02/02/relaxing-acr-storage-limits-with-tools-to-self-manage/) a while back, we were still showing a donut for the current amount of storage. 
+We've heard customers tell us that vulnerability scanning is table stakes for container registries. We agree. With partners like Aqua and TwistLock, customers can get the must-haves complete, today. ACR provides launch points to the Azure Marketplace, helping customers integrate vulnerability scanning.
 
-![](./media/storage-donut.png)
+Starting fall '19, [Azure Security Center][asc] will start offering registry scanning and runtime protection. As with any new offering, it will take time for us to dial in the experiences.
 
+We believe customers should have a choice for their security scanning solutions and will continue to provide extensibility, just as users can choose the security scanning of their users machines and their servers.
 
-This was just confusing, and we joked about making it an overflowing cupcake. We settled on simply showing just the facts:
+### Limit Endpoint Access /VNet Support
 
-![](./media/storage-non-cupcake.png)
+Customers have asked for limitations on their registry, based on the IP, not just authentication. As a shared registry API, this does present some challenges that we're incrementally addressing.
```

Comment:
Don't know if limitations can be taken as positive. I'm guessing the idea here is that customers want more granular access control in terms of who and how the registry is accessed.